packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -104,15 +104,22 @@ export interface IHierBlock extends IMergeBlock {
 }
 
 export interface IRemovalInfo {
-    removedSeq?: number;
-    removedClientId?: number;
-    removedClientOverlap?: number[];
+    removedSeq: number;
+    removedClientIds: number[];
+}
+export function toRemovalInfo(maybe: Partial<IRemovalInfo> | undefined): IRemovalInfo | undefined{
+
```

Comment:
If maybe is a Partial instance, is it possible for just one of removedClientIds and removedSeq to be undefined? Is it possible for maybe to be passed as "IRemovalInfo | undefined"?

Reply:
this function will ensure either none or both at set. generally, what we pass in is ISegment, which extends `Partial<IRemovalInfo>` as it only sets those on removal. in the future with more refactorings we may be able to tighten the typing but i think right now this is the best we can do.

Reply:
I see. Thanks.